generate.sh

```diff
@@ -0,0 +1,29 @@
+mkdir temp
+
+#cp rep/doc temp/
+
+cd temp
+sphinx-quickstart -q -v $VERSION -a $AUTHOR -p $PROJECTNAME
+rm index.rst
+cd ..
+cp -a $DOCPATH. temp/
+cd temp
+make html
+git config --global user.email $EMAIL
```

Comment:
Since this is gonna be an automated push by a machine, we do not want the commits linked to a specific person. Use 
```bash
git config --global user.name "Travis CI"
git config --global user.email "noreply+travis@fossasia.org"
```